src/sections/Books-grid/index.js

```diff
@@ -41,28 +40,60 @@ const BooksGrid = ({hide_path, limit}) => {
     );
 
     let path = hide_path ? "" : "Books";
-    let no_of_items = limit ? limit : data.length;
```

Comment:
This prop was passed down from pages/learn/index.js. Can you remove that also?

Reply:
I checked, it was never passed 😆